config/webpack.config.js

```diff
@@ -197,9 +197,16 @@ function generateHtmlFiles(buildPath, scaffoldAssets) {
     template = {},
     widgetType,
     widgetTemplate,
+    useLocalStylesAndComponents,
   }) =>
     new HtmlPlugin({
-      chunks: ['polyfills', 'web-components', 'vendor', 'style', entryName],
+      chunks: [
+        'polyfills',
+        useLocalStylesAndComponents ? null : 'web-components',
+        'vendor',
+        useLocalStylesAndComponents ? null : 'style',
+        entryName,
+      ],
```

Comment:
This change will only affect local development when running vets-website. If an app's registry entry has `useLocalStylesAndComponents` set to true, then we don't want to include the `web-components` entry file on the HTML page that webpack generates in order to emulate what we would get from the content-build.